tests/snuba/sessions/test_sessions.py

```diff
@@ -568,7 +567,7 @@ def test_fetching_release_sessions_time_bounds_for_different_release_with_no_ses
 
 class SnubaSessionsTestMetrics(ReleaseHealthMetricsTestCase, SnubaSessionsTest):
     """
-    Same tests as in SnunbaSessionsTest but using the Metrics backendx
+    Same tests as in SnunbaSessionsTest but using the Metrics backend
     """
 
     pass
```

Comment:
Does this indicate that tests will be added in the future for metrics? If so I would clarify that with a `ToDo` comment

Reply:
This class runs the same tests with `SnubaSessionsTest` from which is derived but with the metrics backend due to it being derived from `ReleaseHealthMetricsTestCase` , no further code needed

Reply:
Oh okay, I did not get that from the comment .. I thought you were just using the class as a placeholder to write future tests.. maybe `Runs the same tests as in SnubaSessionsTestClass but using the Metrics backend`